README.md

```diff
@@ -313,8 +319,9 @@ func.call(obj, val)
 **Babylon Label**: [Spec: BigInt](https://github.com/babel/babylon/issues?utf8=%E2%9C%93&q=label%3A%22Spec%3A%20BigInt%22%20)  
 <details>
 <summary>Code Example</summary>
-```js
-1n; // integer
+
+```
```

Comment:
@Kovensky added them back. 

I don't have push rights, so if you're happy with what's there so far, and don't mind, would you please merge this PR? (and i'll do another batch in a different one). That way I don't need to repeatedly rebase if somebody makes updates to `master`! :)